authorization_services/topics/policy-regex-policy.adoc

```diff
@@ -0,0 +1,31 @@
+[[_policy_regex]]
+= Regex-Based Policy
+
+You can use this type of policy to define regex conditions for your permissions.
+
+To create a new regex-based policy, select *Regex* in the dropdown list in the upper right corner of the policy listing.
```

Comment:
We now call it an "item list" - IBM/Red Hat standards.

Reply:
@andymunro I changed "dropdown list" to "item list".